src/components/Exchange/PositionEditor.js

```diff
@@ -204,8 +211,8 @@ export default function PositionEditor(props) {
       return t`Min leverage: 1.1x`;
     }
 
-    if (nextLeverageExcludingPnl && nextLeverageExcludingPnl.gt(30.5 * BASIS_POINTS_DIVISOR)) {
-      return t`Max leverage: 30x`;
+    if (nextLeverage && nextLeverage.gt(MAX_INITIAL_LEVERAGE)) {
+      return t`Max leverage: ${(MAX_INITIAL_LEVERAGE / BASIS_POINTS_DIVISOR).toFixed(1)}x`;
```

Comment:
Why is it called the "INITIAL" leverage?)

Reply:
just wondering

Reply:
i not sure i understand neither. it was MAX_LEVERAGE, is there a reason to use "initial"?